kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/GCSBuilder.java

```diff
@@ -36,39 +37,53 @@
   private static final Logger logger = LoggerFactory.getLogger(GCSBuilder.class);
 
   private final String projectName;
-  private String keyFileName;
+  private String keyFile;
+  private String keyFileType;
 
-  public GCSBuilder(String projectName) {
+  public GCSBuilder(String projectName, String keyFileType) {
```

Comment:
we should add `setKeyFileType` rather than adding an arg to the constructor to preserve the existing pattern.